kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-runtime/pom.xml

```diff
@@ -831,6 +831,20 @@
               </target>
             </configuration>
           </execution>
+          <execution>
+            <id>Remove unused appformer-js-monaco assets</id>
+            <phase>install</phase>
```

Comment:
is the `install` phase right? shouldn't we execute it during a previous phase?

Reply:
Actually, we need to execute at this phase. As `monaco.min.js` comes from the `appformer-js-monaco` dependency, it doesn't exist at the `process-resources` phase.